packages/react-devtools-shared/src/backend/renderer.js

```diff
@@ -1247,6 +1254,88 @@ export function attach(
     }
   }
 
+  function getPerfInsight(
+    prevFiber: Fiber | null,
+    nextFiber: Fiber,
+  ): PerfInsight | null {
+    switch (getElementTypeForFiber(nextFiber)) {
+      case ElementTypeClass:
+      case ElementTypeFunction:
+      case ElementTypeMemo:
+      case ElementTypeForwardRef:
+        if (prevFiber === null) {
+          return {
+            isFirstMount: true,
+            isContextEqual: false,
+            isContextDeepEqual: false,
+            didHooksChange: false,
+            arePropsEqual: false,
```

Comment:
Some of these props are named "did X change" and some are named "is X equal". What do you think about naming everything in the "did X change" format? That way the initial mount wouldn't be in a conflicting state of `isFirstMount: true` and `arePropsEqual: false`?

Reply:
Done